canonical-websites/www.ubuntu.com
# 1579 server overview

## Done

* update /server overview to vbt

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/server)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/3.%20Server?preview=server-overview.png) and the [demo](http://www.ubuntu.com-1579-server-overview.demo.haus/server)



## Issue / Card

Fixes #1579
